plugins/woocommerce/src/Internal/ComingSoon/ComingSoonRequestHandler.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Automattic\WooCommerce\Internal\ComingSoon;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+
+/**
+ * Handles the parse_request hook to determine whether the current page needs
+ * to be replaced with a comiing soon screen.
+ */
+class ComingSoonRequestHandler {
+
+	/**
+	 * NHelper dependency.
+	 *
+	 * @var string
```

Comment:
Should be `ComingSoonHelper`?

```suggestion
	 * Coming soon helper.
	 *
	 * @var ComingSoonHelper
```